src/styles/components/StyledAntdPicker.js

```diff
@@ -44,10 +44,30 @@ const dropdownStyles = () => {
     // Background for selected cells
     .ant-picker-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single)::before,
     .ant-picker-cell-in-view.ant-picker-cell-range-end:not(.ant-picker-cell-range-end-single)::before {
-      background: ${props => lighten(0.5, props.theme.primary)};
+      background: ${props => lighten(0.5, props.theme.primary)} !important;
     }
     .ant-picker-cell-in-view.ant-picker-cell-in-range::before {
-      background: ${props => lighten(0.5, props.theme.primary)};
+      background: ${props => lighten(0.5, props.theme.primary)} !important;
+    }
+
+    //Backgrounds for week period
+    :where(.css-dev-only-do-not-override-w8mnev).ant-picker-dropdown
```

Comment:
Did we check if this CSS classes [.css-dev-only] affects in any way when we use the artifact generated?
I did a quick check and I don't know if it can happen something like this: https://github.com/ant-design/ant-design/discussions/38753